artiq/frontend/artiq_master.py

```diff
@@ -154,6 +156,11 @@ def ccb_issue(service, *args, **kwargs):
         bind, args.port_logging))
     atexit_register_coroutine(server_logging.stop)
 
+    core_host, core_port = device_db.get('core')['arguments']['host'], 1383
```

Comment:
Do not crash the master if there is no core device, just print a warning and disable the proxy.

Reply:
The dashboard is also able to respond to dynamic changes to the device database while the master is running. This code is not.